includes/ApplHandlingByID.php

```diff
@@ -434,11 +436,13 @@ function forwardAcceptor ( $id, $type, $user, $next )
 				die();
 			}
 			
+			$next = str_replace('.','$',$next);
+			$next = str_replace(' ','_',$next);
 			$STMT  = $sqlconn->prepare ( "INSERT INTO $next VALUES ( ?,?,? )" );
 			$STMT->bind_param ( 'sss', $id, $type, $row[1] );
 			if ( ! $STMT->execute() )
 			{
-				echo " Unable to perform the task, internal server error.<br>";
+				echo " Unable to forward. Please confirm POST name.<br>";
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->